app/Tenant.php

```diff
@@ -189,4 +189,8 @@ private function getPaystackPayload(array $options = []) {
 
     return $payload;
   }
+
+   public function has_bank_account(){
```

Comment:
this needs to be an [accessor](https://laravel.com/docs/7.x/eloquent-mutators#accessors-and-mutators) so we can officially append it to the tenant model when needed it will ultimately parse to `has_bank_account`

```suggestion
   public function getHasBankAccountAttribute() {
```